fastly/block_fastly_service_package.go

```diff
@@ -34,17 +35,24 @@ func (h *PackageServiceAttributeHandler) Register(s *schema.Resource) error {
 		MinItems:    1,
 		Elem: &schema.Resource{
 			Schema: map[string]*schema.Schema{
+				"content": {
+					Type:          schema.TypeString,
+					Optional:      true,
+					Description:   "The contents of the Wasm deployment package as a base64 encoded string (e.g. could be provided using an input variable or via external data source output variable). Conflicts with `filename`. Exactly one of these two arguments must be specified",
+					ConflictsWith: []string{"package.0.filename"},
```

Comment:
`ConflictsWith` would pass validation if neither `content` or `filename` are set.. maybe better to go with `ExactlyOneOf`? 

https://github.com/hashicorp/terraform-plugin-sdk/blob/main/helper/schema/schema.go#LL284C2-L284C2